geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/AbstractGeoIngestProcessor.scala

```diff
@@ -415,8 +414,7 @@ object AbstractGeoIngestProcessor {
           .name("ConverterName")
           .required(false)
           .description("Choose an SimpleFeature Converter defined by a GeoMesa SFT Provider (preferred)")
-          .allowableValues(ConverterConfigLoader.listConverterNames.sorted: _*)
```

Comment:
I assumed `allowableValues` would perform validation but i guess not... is there any reason to remove it though? I think it makes it a drop down instead of free text (may be wrong there)

Reply:
It did perform validation, but expressions like ${THE_SFT}, which are not in the list of allowed values are considered invalid. The presence of ".expressionLanguageSupported(..." seems to have overridden it: the drop-down is no-longer there. See khobbs:8080/nifi

Reply:
is that nifi running with these code changes or without them?

Reply:
It appears that allowable values are not compatible with expression language support: https://issues.apache.org/jira/browse/NIFI-1832
I'm not crazy about not having it be a drop-down - I think that is going to be a pain for most people. @jahhulbert-ccri @jnh5y what do you guys think?

Reply:
I think having the drop down would be preferable.  Are there any other options here?  E.g., either provide a nifiExpressionConverterName or selecting something from the drop down?

Reply:
hmm i assumed drop down is preferable too from an easy of use perspective...I guess I could reconsider but my default position are that the users of it would like to see the list of things available...We could always change the OTHER sft name and convertername to use expression language and pull it from there?

Reply:
@jahhulbert-ccri I like it.  Does that make sense as an option to everyone else?

Reply:
which fields are you referring to? we have feature name override, but that by definition does not match any of the dropdown names. And there is no other converter name field. Are you suggesting adding two new fields?